sdks/java/io/common/src/test/java/org/apache/beam/sdk/io/common/IOTestPipelineOptions.java

```diff
@@ -88,4 +88,13 @@
   Integer getCassandraPort();
   void setCassandraPort(Integer port);
 
+  /*
+   * Options for test pipeline for TextIO in
```

Comment:
<!--new_thread; commit:9c483a5dd68450cfc0d8bf1cfdd151961fa6cdaa; resolved:0-->
Options for test pipeline for file-based I/O in 'sdks/java/io/file-based-io-tests/'.

Reply:
done (sorry, missed this before)